consumers/email-consumer/tests/test_email_consumer.py

```
@@ -0,0 +1,97 @@
+import os
+import pytest
+import requests
+from unittest.mock import patch, MagicMock
+
+from email_consumer import EmailConsumer
+
+
+@pytest.fixture
+def email_consumer():
```

Ruff (0.8.2)

Missing return type annotation for public function `email_consumer`

(ANN201)